mountinfo/mountinfo_linux.go

```diff
@@ -18,11 +18,9 @@ import (
 func GetMountsFromReader(r io.Reader, filter FilterFunc) ([]*Info, error) {
 	s := bufio.NewScanner(r)
 	out := []*Info{}
-	var err error
 	for s.Scan() {
-		if err = s.Err(); err != nil {
```

Comment:
That's a curse that I have removed this from so many projects but failed to do so for this one :)

Reply:
Does golang-ci-lint detect these?

Reply:
I guess not, otherwise that'd be fixed earlier